Assignment_one.py

```diff
@@ -1,26 +1,25 @@
 import requests
-import json
 
-base_url = "https://automationexercise.com/api/productsList"
+url = "https://automationexercise.com/api/productsList"
 
-Response = requests.get(base_url)
-# print(Response)
+response = requests.get(url)
 
-#Validating the content and status code
-code = Response.status_code
-print("Status code is ", code)
-assert code == 200 ,"Code doesn't match "
-print(json.dumps(Response.json(), indent=4))
-json_resp = Response.json()
-
-#Return List
-List = list(Response)
-print(List)
+# Validate status code
```

Comment:
use framework to manage and run the cases

Reply:
Made the changes, Kindly verify it.